proto/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: test all compile clean
+
+PROTOC := $(shell which pbjs)
+
+JS_PROTOS = alc_health.piqi_pb.js common_proto.piqi_pb.js dict.piqi_pb.js host_metadata.piqi_pb.js 
```

Comment:
if it is possible to not include compiled pb js (azure can compile it by itself), please remove it.

Reply:
I was thinking of making a `postinstall` npm action but this will require too many magical scripting and looks like it doesn't worth it. 
It is not guaranteed that the build machine has `binutils` or `make` installed - this is especially true for Azure where Azure deployment runs `npm install` on its own, potentially on Windows machine.
Anyway even if we have the postinstall script which compiles protobufs it still requires attention every time new protobuf file is added.
In general I think checking in precompiled protobufs is a safer and easier option for our use cases for both AWS Lambda and Azure Functions.